source/common/modules/markdown-editor/plugins/render-mermaid.js

```diff
@@ -27,8 +27,7 @@ const mermaid = require('mermaid')
   'use strict'
 
   // Initialise the mermaid API
-  // TODO: Theming!
-  mermaid.mermaidAPI.initialize({ startOnLoad: false, theme: 'dark'/*, theme: null */ })
+  // mermaid.mermaidAPI.initialize({ startOnLoad: false, theme: 'base'})
```

Comment:
You can remove this part completely, since you've effectively copied it to after you've extracted the necessary styles!